src/core/vaults/RVault.sol

```diff
@@ -22,6 +22,12 @@ contract RVault is RVaultBase {
     /// @dev Version of RVault implementation
     string public constant VERSION = '1.0.0';
 
+    /// @inheritdoc IVault
+    uint public constant UNIQUE_INIT_ADDRESSES = 1;
```

Comment:
use internal constant and public getter with overriding of vault base empty getter